sevencooks-gmbh-co-kg/eslint-config-sevencooks
# feat(base): add no-console rule (again)

- Updated dependencies (some were really old)
- added rule `no-console` again (it was removed from the recommended): https://eslint.org/docs/rules/

**Info: the additional errors occurring due to the dependency updates will be fixed by a PR in the corresponding repository. This is no reason to complain or request changes. It's totally common and we already did this several times before. It's just more time ago.**

### How to test
- open `weekly-planner` project
- replace `eslint-config-sevencooks` version with: `sevencooks-gmbh-co-kg/eslint-config-sevencooks#update`
- `rm -rf node_modules && npm i`
- `npm run lint` (there may be some more errors due to updated dependencies)
- `no-console` should be an error again
